node_modules/path-is-absolute/package.json

```diff
@@ -21,7 +21,7 @@
   "_resolved": "https://registry.npmjs.org/path-is-absolute/-/path-is-absolute-1.0.1.tgz",
   "_shasum": "174b9268735534ffbc7ace6bf53a5a9e1b5c5f5f",
   "_spec": "path-is-absolute@^1.0.0",
-  "_where": "/Users/gautamb/Documents/test-files/synopsys-intelligent-scan/node_modules/glob",
```

Comment:
Is there any specific reason to change the path here?

Reply:
This is done by `npm install`, the changes are only related to node_modules, these changes will not be effecting the IO action workflow.

Reply:
Ok got it. Thanks